### PR TITLE
feat(llama): Add Llamapay Module

### PR DESCRIPTION
**Pull request by tomwade**
## Description

This PR adds a module for the Llamapay system. In a similar way to the Sablier module, this pulls through the current withdrawable amount of each token currently being streamed to an address through the platform. This module was heavily inspired by the Sablier code structure and approach.

## Checklist

- [X] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [X] As a contributor, my Ethereum address/ENS is: `twade.eth`
- [x] As a contributor, my Twitter handle is: `tomwade`

## How to test?

Can currently be tested with `0x0f294726A2E3817529254F81e0C195b6cd0C834f` which provides a WETH stream.

### CODE REVIEW

File: src/apps/llamapay/llamapay.definition.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  keywords: [],
+  links: {},
+
+  supportedNetworks: {
+    [Network.ETHEREUM_MAINNET]: [AppAction.VIEW],
+    [Network.POLYGON_MAINNET]: [AppAction.VIEW],
+    [Network.OPTIMISM_MAINNET]: [AppAction.VIEW],
+    [Network.FANTOM_OPERA_MAINNET]: [AppAction.VIEW],
+    [Network.AVALANCHE_MAINNET]: [AppAction.VIEW],
+    [Network.ARBITRUM_MAINNET]: [AppAction.VIEW],
```
Comment (immasandwich) [inline src/apps/llamapay/llamapay.definition.ts:30]: Can you remove everything except for Ethereum here?

File: src/apps/llamapay/llamapay.definition.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const LLAMAPAY_DEFINITION = appDefinition({
+  id: 'llamapay',
+  name: 'Llamapay',
+  description: 'LlamaPay integration for Zapper',
```
Comment (immasandwich) [inline src/apps/llamapay/llamapay.definition.ts:9]: `LlamaPay is a multi-chain protocol that allows you to automate transactions and stream them by the second.` would be better!

**Review by immasandwich — APPROVED** [review]
Nice, 2 small comments then let's ship it.

**Comment by tomwade** [comment]
@immasandwich Good points! Updated against both 👍

File: src/apps/llamapay/llamapay.definition.ts (hunk 1 above)
Comment (tomwade) [reply src/apps/llamapay/llamapay.definition.ts]: I had been a little optimistic in my first run through 😆

**Pull request merged**